poll/public/handlebars/poll_studio.handlebars

```diff
@@ -0,0 +1,22 @@
+<script id="poll-form-component" type="text/html">
+{{#each items}}
+<li class="field comp-setting-entry is-set poll-{{noun}}-studio-item">
+    <div class="wrapper-comp-setting">
+        <label class="label setting-label poll-setting-label" for="{{noun}}-label-{{key}}">{{noun}}</label>
```

Comment:
if you make the associated ```<input>``` a child of the ```<label>``` you don't have to worry about the ```for/id``` references.